roles/confluent.control_center/tasks/main.yml

```
@@ -243,15 +250,22 @@
     owner: "{{control_center_user}}"
     group: "{{control_center_group}}"
     mode: 0640
+  when: privileged_install|bool
 
 - name: Write Service Overrides
   template:
     src: override.conf.j2
     dest: "{{control_center.systemd_override}}"
-    mode: 0640
-    owner: "{{control_center_user}}"
-    group: "{{control_center_group}}"
+    mode: "{{ omit if not privileged_install else '0640' }}"
+    owner: "{{ omit if not privileged_install else control_center_user }}"
+    group: "{{ omit if not privileged_install else control_center_group }}"
+  notify: restart control center
+
+- name: Install Service in User-mode
+  shell:
+    cmd: "XDG_RUNTIME_DIR=/run/user/$(id -u) systemctl --user enable {{control_center.systemd_file}}"
```

what is the significance of 
```
shell:
  cmd:
```
? Elsewhere in the code we typically do:
```suggestion
shell: |
  XDG_RUNTIME_DIR=/run/user/$(id -u) systemctl --user enable {{control_center.systemd_file}}
```

---

from ansible's doc it looks like it's the same